README.md

```diff
@@ -16,7 +16,7 @@
 
 ## Demo
 
-To view a demo online: <https://vue-datepicker.netlify.com/>
```

Comment:
hmmm you removed alot in README. 

Rollback this file.

Reply:
ok, it's just because i'm using some "markdown beautifier" when trying to add about this new prop